test/integration/puppetlabs/services/jruby/jruby_pool_int_test.clj

```diff
@@ -23,32 +22,20 @@
             [puppetlabs.services.protocols.request-handler :as handler]
             [puppetlabs.services.request-handler.request-handler-core :as handler-core]
             [puppetlabs.ssl-utils.core :as ssl-utils]
-            [puppetlabs.kitchensink.testutils :as ks-testutils]))
+            [puppetlabs.kitchensink.testutils :as ks-testutils]
+            [puppetlabs.testutils :as testutils :refer
```

Comment:
The impression I've gotten is that we want to avoid using `:refer` unless we're referring things from the namespace we're explicitly testing to make it more clear where things are coming from (although I may be totally wrong on that). Saying that, we're fairly inconsistent about when we do and don't use `:refer`, so I'm fine with leaving these `:refer`s as-is.